java/ql/integration-tests/qlpack.yml

```diff
@@ -0,0 +1,7 @@
+name: codeql/java-integration-tests
```

Comment:
Make this pack anonymous so that it can be copied into multiple test folders without conflicts.
```suggestion
```